contracts/libraries/RewardLib.sol

```diff
@@ -18,28 +18,14 @@ library RewardLib {
     int32 ratingToReward;
   }
 
-  struct StatusRewardContainer {
-    mapping(address => mapping(uint16 => StatusReward)) statusReward;
-  }
-
-  struct StatusReward {
-    bool isPaid;
-  }
-
   struct UserRewards {
-    mapping(address => mapping(uint16 => PeriodRating)) userRewards;
-  }
-
-  /// @notice Get information about the user's reward
-  /// @param userRewards The mapping containing all user's rewards
-  /// @param user The use who has rewards
-  /// @param period The period which we get reward
-  function getUserPeriodRating(RewardLib.UserRewards storage userRewards, address user, uint16 period) internal view returns (RewardLib.PeriodRating storage) {
-    return userRewards.userRewards[user][period];
+    uint32[] activeInCommunity;
+    bool isPaid;
```

Comment:
Remove from here. Return back `StatusReward`. `isPaid` has to be updated on the side of token contract.